recipes-wpe/wpewebkit/wpewebkit_2.28.bb

```diff
@@ -1,5 +1,6 @@
 
 require wpewebkit.inc
+DEPENDS_append = " libsoup-3.0"
```

Comment:
If the 2.28 requires higher glib, then it should be present here as the dependency with version

Reply:
Is it required to specify the version ? Currently buildroot also using same version for both libsoup-2.7. and lib soup-3